apps/src/templates/progress/GoogleClassroomAttributionLabel.jsx

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import {connect} from 'react-redux';
+import {canShowGoogleShareButton} from './googlePlatformApiRedux';
+
+const style = {
+  textAlign: 'center',
+  paddingTop: 5
+};
+
+class GoogleClassroomAttributionLabel extends React.Component {
+  static propTypes = {
+    // redux provided
+    hidden: PropTypes.bool.isRequired
+  };
+
+  render() {
+    if (!this.props.hidden) {
+      return <p style={style}>{i18n.googleClassroomAttribution()}</p>;
+    } else {
+      return null;
+    }
+  }
+}
+
+export default connect(state => ({
+  hidden: !canShowGoogleShareButton(state)
```

Comment:
Rather than negating this here and on line 19, would it make more sense to name this `visible` instead of `hidden`?

Reply:
hahahaha, thank you!! i didn't like negating it but i couldn't think of a good word to mean the opposite (i got stuck on "show", which i didn't like). `visible`, obviously!!